saved-sites/saved-sites-impl/src/main/java/com/duckduckgo/savedsites/impl/sync/SavedSitesSyncDataObserver.kt

```diff
@@ -67,7 +69,7 @@ class SavedSitesSyncDataObserver @Inject constructor(
         if (!dataObserverJob.isActive) {
             dataObserverJob += coroutineScope.launch(dispatchers.io()) {
                 Timber.d("Sync-Feature: Listening for changes to Saved Sites")
-                savedSitesRepository.lastModified().collect {
+                savedSitesRepository.lastModified().drop(1).collect {
```

Comment:
Can you throw a comment as to **why** the `drop(1)`. We will forget in couple weeks why we added it